src/cursor/cur_table.c

```diff
@@ -512,6 +519,7 @@ __curtable_search_near(WT_CURSOR *cursor, int *exact)
 
 err:
     CURSOR_REPOSITION_END(cursor, session);
+    API_RETRYABLE_END(session, ret);
```

Comment:
These pairs appear to be carefully placed in between other macros. Maybe some comments about why it is here that get copied/repeated to help others understand in the future.

Reply:
@sueloverso I'm not in favor of putting comments here.  In order to be understandable, I think they'd have to be kind of long, which feels a little ugly for mostly boilerplate code.  Effectively what they'd say (I think) is you need to "balance" the begins with the ends for all these API macros, and I added that to the comment for the macro definitions.  See what you think of that wording, if it seems enough.  BTW, if you order them wrong, you generally get loads of compilation errors.